tests_integration/nativeapp/test_version.py

```diff
@@ -377,6 +377,8 @@ def test_nativeapp_version_create_package_no_magic_comment(
             row.pop("created_on", None)
             # Remove metric_level field
             row.pop("metric_level", None)
+            # Remove auditlog_level field
+            row.pop("auditlog_level", "OFF")
```

Comment:
How about creating a minimal set of required fields for this snapshot to work and discard all others.
Another option is to create a tuple of fields to discard.

With first case we won't be notice if any new field is added - only removed ones will trigger test to fail
With second we will need to update each time a new field is added.

Finally, do we really need to test for payload in audit log? Do we control the content?
Maybe we could only check if response exists and/or response status?

Reply:
Changed